modAionBase/src/org/aion/base/type/ITransaction.java

```diff
@@ -1,48 +1,22 @@
 package org.aion.base.type;
 
 import java.math.BigInteger;
+import org.aion.vm.api.interfaces.TransactionInterface;
 
 /** @author jin */
-public interface ITransaction extends Cloneable {
-
-    byte[] getHash();
+public interface ITransaction extends Cloneable, TransactionInterface {
```

Comment:
The ITransaction/TransactionInterface thing is a bit confusing, but as we talked about in-person, it's ok as like a intermediate step that will be eventually refactored.  Could you just either add a javadoc or TODO comment here or make a Github/Jira issue about this so we remember to revisit some time in the near future?